strapi/strapi
# feat: New command 'components:list'

### What does it do?

It introduces a new CLI command:

```
strapi components:list
```

### Why is it needed?

Similar to the `content-types:list` command it can be handy to quickly get a list of all the available components in your Strapi installation.

### How to test it?

1. Install the PR
2. Setup your Strapi app
3. Add some components
4. Run the command and see a list of the components
